torch/onnx/_constants.py

```diff
@@ -4,8 +4,8 @@
 
 ONNX_BASE_OPSET = 9
 ONNX_MIN_OPSET = 7
-ONNX_MAX_OPSET = 18
-ONNX_TORCHSCRIPT_EXPORTER_MAX_OPSET = 17
+ONNX_MAX_OPSET = 19
+ONNX_TORCHSCRIPT_EXPORTER_MAX_OPSET = 19
```

Comment:
```suggestion
ONNX_TORCHSCRIPT_EXPORTER_MAX_OPSET = 17
```

Reply:
This is used by https://github.com/justinchuby/pytorch/blob/63919ed8a2114e5e8f80c93247acabb5d7b71bf8/torch/onnx/utils.py#L1528-L1539 to emit warnings